templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -57,11 +63,11 @@
 </span>
 
 {% if users_id_editor > 0 and date_creation != date_mod %}
-   <span class="badge text-wrap ms-1 d-none d-md-block">
+   <span class="badge user-select text-wrap ms-1 d-none d-md-block">
```

Comment:
There is already a `user-select-auto` class defined by bootstrap.

Reply:
done